### DISCUSSION TITLE
error CSV agent

### DISCUSSION THREAD

**Discussion by monroyhd** [FlowiseAI/Flowise#649] [General]
PythonError: Traceback (most recent call last): File "/lib/python311.zip/_pyodide/_base.py", line 540, in eval_code_async await CodeRunner( File "/lib/python311.zip/_pyodide/_base.py", line 363, in run_async coroutine = eval(self.code, globals, locals) ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ File "<exec>", line 10, in <module> UnicodeDecodeError: 'utf-8' codec can't decode byte 0xd5 in position 72: invalid continuation byte

**Comment by chungyau97** [comment]
Hi @monroyhd,

Are you using `ChatOpenAI` node?

**Reply by sabatale** [reply]
Trying to use this with Cohere always returns:

`Error: Sorry, I'm unable to find answer for question`
